x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents.tsx

```diff
@@ -6,23 +6,45 @@
 
 import React from 'react';
 
-import {
-  EuiPageHeader,
-  EuiPageHeaderSection,
-  EuiTitle,
-  EuiPageContent,
-  EuiPageContentBody,
-} from '@elastic/eui';
+import { EuiPageHeader, EuiPageHeaderSection, EuiTitle, EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { useValues } from 'kea';
+import { i18n } from '@kbn/i18n';
 
+import { DocumentCreationButton } from './document_creation_button';
 import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { FlashMessages } from '../../../shared/flash_messages';
 import { DOCUMENTS_TITLE } from './constants';
+import { EngineLogic } from '../engine';
+import { AppLogic } from '../../app_logic';
+import { SearchExperience } from './search_experience';
+
+const MetaEngineCallout: React.FC = () => (
+  <>
+    <EuiCallOut
+      iconType="iInCircle"
+      title={i18n.translate('xpack.enterpriseSearch.appSearch.documents.metaEngineCallout.title', {
+        defaultMessage: 'You are within a Meta Engine.',
```

Comment:
Not to open the copy can of worms, but should Meta Engine be title-cased I wonder? Maybe worth checking with Chris later on when we do a pass

Reply:
True.